### DISCUSSION TITLE
Space-p and Space-R not working inside Tmux

### DISCUSSION THREAD

**Discussion by yusuphgammut** [helix-editor/helix#8470] [General]
Hello.

I recently updated my operating system to macOs Ventura and since then the Space-p and Space-R keymaps are not working inside Tmux. However, Ctrl-v works as expected. Before that they worked ok. Any idea on how to address this?

Alacritty version: `alacritty 0.13.0-dev (e35e5ad)`.
Tmux version: `tmux 3.3a`.
Helix version: `helix 23.05 (072e1eae)`.

Outside Tmux, `hx --health` returns this:
```
Clipboard provider: pbpaste+pbcopy
System clipboard provider: pbpaste+pbcopy
```

Inside Tmux, `hx --health` returns this:
```
Clipboard provider: tmux
System clipboard provider: tmux
```

And `tmux show -s set-clipboard` returns this:
```
set-clipboard external
```

**Comment by HellConnector** [comment]
I was able to "fix" this with this simple line in my `.zshrc`, but I think it is completely unnatural and I don't really know how it might affect tmux's behavior.
```
# temporary fix for clipboard provider in Helix
unset TMUX
```

**Comment by archseer** [comment]
Created an issue out of this discussion

**Reply by yusuphgammut** [reply]
Thanks!

**Comment by Agilan989** [comment]
Noted
Thank you
Best regards
Agilanberan
